src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/NovelAdjacencyReferenceLocations.java

```diff
@@ -248,38 +258,44 @@ private static boolean isFirstInPartner(final ChimericAlignment ca, final SAMSeq
         }
     }
 
+    private static final class BreakpointsInferenceFactory {
```

Comment:
Oh, sorry, I didn't notice this. I don't think you need a separate class to hold this method, you could just make this a static method on the containing class, unless you prefer it this way.

Reply:
Will be changed in the final merge. Thanks!